### PR TITLE
OPENTOK-43278: Replace bintray links with Maven Central

### PR DESCRIPTION
**Pull request by jintgeorge**

### CODE REVIEW

File: Archiving/build.gradle [hunk 1]
```diff
@@ -18,6 +18,7 @@ allprojects {
         jcenter()
         repositories {
             jcenter()
+            mavenCentral()
             maven { url 'https://maven.google.com' }
             maven { url 'https://tokbox.bintray.com/maven' }
         }
```
Comment (v-kpheng) [inline Archiving/build.gradle:24]: bintray is being sunset so users won't be able to download artifacts from there anymore.  Should bintray links still be included?

**Review by v-kpheng — COMMENTED** [review]
Looks good! 💪 

That said, left a question.  Thanks!

**Comment by jintgeorge** [comment]
Need to create another PR with updates master branch.

**Comment by igorwojda** [comment]
FYI: We still have to do a bit of migration, however, we need to wait for external lib (easypermissions) to be migrated
https://github.com/opentok/opentok-android-sdk-samples/pull/380

**Pull request closed without merging**